### PR TITLE
Change http to https

### PR DESCRIPTION
**Pull request by skasturi**
Switches all http links to https. 

Ref issue: https://github.com/Microsoft/NimbusML/issues/57

### CODE REVIEW

File: src/python/docs/docstrings/FastLinearClassifier.txt [hunk 1]
```diff
@@ -56,7 +56,7 @@
             content/uploads/2016/06/main-3.pdf>`_
     
             `Stochastic Dual Coordinate Ascent Methods for Regularized Loss
-            Minimization <http://www.jmlr.org/papers/volume14/shalev-
+            Minimization <https://www.jmlr.org/papers/volume14/shalev-
```
Comment (skasturi) [inline src/python/docs/docstrings/FastLinearClassifier.txt]: Broken link

File: src/python/docs/docstrings/FastLinearRegressor.txt [hunk 2]
```diff
@@ -56,7 +56,7 @@
             content/uploads/2016/06/main-3.pdf>`_
     
             `Stochastic Dual Coordinate Ascent Methods for Regularized Loss
-            Minimization <http://www.jmlr.org/papers/volume14/shalev-
+            Minimization <https://www.jmlr.org/papers/volume14/shalev-
```
Comment (justinormont) [inline src/python/docs/docstrings/FastLinearRegressor.txt]: Recommend checking that all the new links work:
```suggestion
            Minimization <http://www.jmlr.org/papers/volume14/shalev-
```

Original works: http://www.jmlr.org/papers/volume14/shalev-shwartz13a/shalev-shwartz13a.pdf

**Comment by ganik** [comment]
Hi @skasturi, thank you for doing this work. Is there anyway you can check all the links are still working after moving to https ? thx

**Comment by skasturi** [comment]
We should be able to automate the check. Let me give it a shot next week sometime. 

The method I am thinking is to extract all urls and run them through a a script to check if they are giving a success code.

**Comment by ganik** [comment]
this has been fixed by #346

**Pull request closed without merging**